src/vs/workbench/parts/terminal/electron-browser/terminalActions.ts

```diff
@@ -56,8 +56,7 @@ export class KillTerminalAction extends Action {
 }
 
 /**
- * Copies the terminal selection. Note that since the command palette takes focus from the terminal,
- * this can only be triggered via a keybinding.
+ * Copies the terminal selection.
```

Comment:
The comment is still relevant as you can't trigger it through the command palette. This is what it should probably say though:

```js
/**
 * Copies the terminal selection. Note that since the command palette takes focus from the terminal,
 * this cannot be triggered through the command palette.
 */
```